clang/include/clang/AST/ASTContext.h

```diff
@@ -1185,17 +1185,22 @@ class ASTContext : public RefCountedBase<ASTContext> {
     return CanQualType::CreateUnsafe(getComplexType((QualType) T));
   }
 
+  /// Get the default pointer interpretation in use.
+  PointerInterpretationKind getDefaultPointerInterpretation() const;
+
   /// Return the uniqued reference to the type for a pointer to
   /// the specified type.
-  QualType getPointerType(QualType T,
-                          PointerInterpretationKind PIK = PIK_Default) const;
+  QualType getPointerType(QualType T) const {
```

Comment:
How much effort would it be to remove this overload? Requiring the PIK argument would have caught e.g. the loss of `__capability` in clang/lib/Sema/SemaExprCXX.cpp that I fixed a while back.

However, that would also result in more merge conflicts so I'm not sure it's worthwhile.

Reply:
There are 423 calls to getPointerType in clang. Most of the time we really do want the default pointer type, so that'd be a lot of churn.